packages/nextjs/next.config.ts

```diff
@@ -3,6 +3,9 @@ import type { NextConfig } from "next";
 const nextConfig: NextConfig = {
   reactStrictMode: true,
   devIndicators: false,
+  images: {
+    domains: ["res.cloudinary.com"],
```

Comment:
There's a warning in the Next.js terminal console regarding `images.domains`. Can you fix it, please?